examples/tilted_bottom_boundary_layer.jl

```diff
@@ -86,179 +65,173 @@ plot(grid.Δzᵃᵃᶜ[1:Nz], grid.zᵃᵃᶜ[1:Nz],
-# oscillations due to an unbalanced resolved flow:
-#
+@inline constant_stratification(x, y, z, t, p) = p.N² * (x * p.ĝ[1] + z * p.ĝ[3])
 
-V_bg(x, y, z, t, p) = p.V∞
-V_field = BackgroundField(V_bg, parameters=(; V∞=params.V∞))
+# is _not_ periodic in ``x``. Thus we cannot explicitly model a constant stratification
+# on an ``x``-periodic grid such as ours. Instead, we simulate periodic
+# _perturbations_ away from the constant density stratification by imposing
+# a constant stratification as a `BackgroundField`,
 
+B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = 1e-5))
 
+# where ``N² = 10⁻⁵ \rm{s}⁻¹`` is the background buoyancy gradient.
 
 # ## Bottom drag
 #
-# We set-up a bottom drag that follows Monin-Obukhov theory. Note that we need to include`V∞`
-# in the velocity, since we will model it as a background field.
-#
+# We impose bottom drag that follows Monin-Obukhov theory.
+# We include the background flow in the drag calculation,
+# which is the only effect the background flow enters the problem,
 
-const κ = 0.4 # von Karman constant
-z₁ = CUDA.@allowscalar znodes(Center, grid)[1] # Closest grid center to the bottom
-cᴰ = (κ / log(z₁/params.z₀))^2 # Drag coefficient
+V∞ = 0.1 # m s⁻¹
+z₀ = 0.1 # m (roughness length)
+κ = 0.4 # von Karman constant
+z₁ = znodes(Center, grid)[1] # Closest grid center to the bottom
+cᴰ = (κ / log(z₁ / z₀))^2 # Drag coefficient
 
 @inline drag_u(x, y, t, u, v, p) = - p.cᴰ * √(u^2 + (v + p.V∞)^2) * u
 @inline drag_v(x, y, t, u, v, p) = - p.cᴰ * √(u^2 + (v + p.V∞)^2) * (v + p.V∞)
 
-drag_bc_u = FluxBoundaryCondition(drag_u, field_dependencies=(:u, :v), parameters=(cᴰ=cᴰ, V∞=params.V∞))
-drag_bc_v = FluxBoundaryCondition(drag_v, field_dependencies=(:u, :v), parameters=(cᴰ=cᴰ, V∞=params.V∞))
+drag_bc_u = FluxBoundaryCondition(drag_u, field_dependencies=(:u, :v), parameters=(; cᴰ, V∞))
+drag_bc_v = FluxBoundaryCondition(drag_v, field_dependencies=(:u, :v), parameters=(; cᴰ, V∞))
 
 u_bcs = FieldBoundaryConditions(bottom = drag_bc_u)
 v_bcs = FieldBoundaryConditions(bottom = drag_bc_v)
 
-
-
 # ## Create the `NonhydrostaticModel`
-#
-# Since the coordinate system is aligned with the tilted domain, we can use a traditional f-plane
-# approximation (only keep the rotation component that's aligned with gravity) by also tilting the
-# rotation from the coordinate system's perpective by `θ_rad` radians using
-# `ConstantCartesianCoriolis` and passing the `rotation_axis` argument:
-#
-
-coriolis = ConstantCartesianCoriolis(f=params.f₀, rotation_axis=ĝ)
-
 #
 # We are now ready to create the model. We create a `NonhydrostaticModel` with an
-# `UpwindBiasedFifthOrder` advection scheme and a `RungeKutta3` timestepper.  For simplicity, we use
-# a constant-diffusivity turbulence closure:
+# `UpwindBiasedFifthOrder` advection scheme, a `RungeKutta3` timestepper,
+# and a constant diffusivity, initialized with a small amount of random noise
```

Comment:
I don't see where the "small amount of random noise" is set further down. Perhaps we drop this statement?

Reply:
Oops. You're right. I had it in there but took it out cause it causes inertial oscillations.

Reply:
```suggestion
# and a constant viscosity and diffusivity.
```